contracts/WithdrawManager.sol

```diff
@@ -32,7 +33,7 @@ contract WithdrawManager {
     ) public {
         tokenRegistry = _tokenRegistry;
         vault = _vault;
-        appID = keccak256(abi.encodePacked(_rollup));
+        appID = keccak256(abi.encodePacked(Rollup(_rollup).appID));
```

Comment:
We may just want to pass in the appID/domain directly here rather than coupling to the Rollup contract.